Branch-SDK-TestBed/AndroidManifest.xml

```diff
@@ -31,18 +31,35 @@
             	<category android:name="android.intent.category.BROWSABLE" />
         	</intent-filter>
         </activity>
-        <activity
-            android:name="io.branch.branchandroiddemo.CreditHistoryActivity">
-        </activity>
-        <activity
-            android:name="io.branch.branchandroiddemo.ReferralCodeActivity">
+
+        <activity android:name="io.branch.branchandroiddemo.CreditHistoryActivity"/>
+
+        <activity android:name="io.branch.branchandroiddemo.ReferralCodeActivity"/>
+
+        <activity android:name=".AutoDeepLinkTestActivity">
+            <!-- Keys for auto deep linking this activity -->
+            <meta-data android:name="io.branch.sdk.auto_link_keys" android:value="auto_deeplink_key_1,auto_deeplink_key_2" />
+            <!-- Optional request ID for launching this activity on auto deep link key matches -->
+            <meta-data android:name="io.branch.sdk.auto_link_request_code" android:value="201" />
```

Comment:
Request code is optional . In most of the case Devs would need an activity result on finishing activity for handling flow differently for auto deep link flow and regular app flow launch. So we need  to `startActivityForResult`. The same  `requestCode` is returned on activity result. It is very common for application to have a base activity which handle `onActivityResult()` and take action depends on the request code. 
   If they don't care about this they don't need  to provide the request code in meta.We are starting the Activity with a default request code.

  Deep link keys are  the actuals keys in the params which this activity need deep linked to